libmscore/marker.cpp

```diff
@@ -107,6 +108,12 @@ void Marker::setMarkerType(Type t)
                   setLabel("coda");
                   break;
 
+            case Type::TOCODASYM:
+                  txt = "To <font size=\"20\"/><sym>coda</sym>";
```

Comment:
Unfortunately we have to hard code the size of that symbol (here to the default size of the 'plain' Coda)